lib/src/components/NxSearchDropdown/NxSearchDropdown.tsx

```diff
@@ -89,6 +89,16 @@ function NxSearchDropdownRender<T extends string | number = string>(
     onSearch(value.trim());
   }
 
+  const handleKeyDown: KeyboardEventHandler = event => {
+    if (event.key === 'Escape') {
+      onSearchTextChange('');
+
+      if (!event.defaultPrevented) {
+        event.preventDefault();
```

Comment:
Why is this needed? Even if it is, is there a point in wrapping it in the conditional?

Reply:
> Why is this needed? Even if it is, is there a point in wrapping it in the conditional?

I guess it would be needed if this was within a modal, to prevent esc from also closing the modal

Reply:
> > Why is this needed? Even if it is, is there a point in wrapping it in the conditional?
> 
> I guess it would be needed if this was within a modal, to prevent esc from also closing the modal

Yeah, the conditional is probably not needed, removed it here: 62efc9de1d1e3d19052e42ae14e214afbab3bcd4